extensions/amp-story/amp-story-grid-layer.md

```diff
@@ -424,11 +424,11 @@ The background preset is mostly useful for designing layouts with both backgroun
 
 ##### Foreground
 
-Use the `2021-foreground` preset for foreground elements. This anchors foreground layers to pin items, such as logos or footnotes, to the page corners or borders.
+Use the `2021-foreground` preset for foreground elements. This layer will always be fully visible and expand to the borders either horizontally or vertically, maximizing the size without cropping. The attribute `anchor` is useful to pin foreground layers to the page corners or borders, such that elements can be positioned relative to those borders or corners (eg: logos or footnotes).
 
 ##### Background
 
-Use the `2021-background` preset for the background layer. This preset covers the entire screen on all mobile devices with the background asset. It does so without letterboxing and keeps scaling consistent with the foreground elements.
+Use the `2021-background` preset for the background layer. This preset covers the entire screen on all mobile phones with the background asset. It does so without letterboxing, and keeps positioning consistent with the foreground elements.
```

Comment:
My only concern when saying things like "all mobile phones" is have we tested on all phones? What about those new Galaxy fold out phones that become double width? A possible alternative:

"This preset covers the entire screen on most mobile devices with the background asset. It does so keeping positioning consistent with the foreground elements. Letterboxing is only needed in extreme cases."

Reply:
Our filter for preventing letterbox on the background layer was "any mobile phone with an OS/ Browser that we support, that is not an experimental screen format as of December 2020". That's not very user friendly to explain and a bit subjective, but leaves out phones like foldables, tablets and watches. Maybe saying "most mobile phones" is just easier 🤷🏼

Reply:
May I suggest the following:

```
Use the `2021-background` preset for the background layer. This preset covers the entire screen on all mobile phones with the background asset. It does so without letterboxing on common screen sizes, and keeps positioning consistent with the foreground elements. 

[tip type="warning"]
In extreme aspect ratio devices, letterboxing can occur. For these cases you may [change the background color in CSS](https://alankent.me/2021/04/23/sizing-large-images-in-web-stories/) for the best visual outcome. 
[/tip]
```